GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx

```diff
@@ -28,6 +50,22 @@ function RegistrationForm() {
             setLokasjon(prevLoc => ({...prevLoc,[name]:val}))
         }
 
+        const onDinnerRadioChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+            setDinnerRadio(e.target.value);
+        } 
+
+        const getDinner = () => {
+            return dinnerRadio === "annet" ? dinnerInput : dinnerRadio;
+        }
+
+        const getDessert = () => {
+            return dessertRadio === "annet" ? dessertInput : dessertRadio;
+        }
+
+        const onDessertRadioChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
An idea to set the dessert input to an empty string when the radio buttons are toggled?